### PR TITLE
FRONTEND: cleanup filenames and constants, add jest setup

### PR DESCRIPTION
**Pull request by swang192**

### CODE REVIEW

File: frontend/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+// For a detailed explanation regarding each configuration property, visit:
```
Comment (Ian-MacLeod) [inline frontend/jest.config.js:1]: Do we want to keep all the commented out settings in this file?

File: frontend/jest.setup.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  useGlobalContext.mockReturnValue(window.store);
+
+  window.mockedRouter = {
+    route: '',
+    pathname: '',
+    query: '',
+    asPath: '',
+    push: jest.fn(() => new Promise(r => r())),
+  };
+  nextRouter.useRouter = () => window.mockedRouter;
```
Comment (Ian-MacLeod) [inline frontend/jest.setup.js:39]: I'm sure that there must be a better way to make these objects available during tests than attaching them to the window, but I don't know what that is offhand. I suppose I'm ok with this for now, but we should look for a nicer solution.

**Review by Ian-MacLeod — COMMENTED** [review]
Looks good

File: frontend/components/Nav.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ import Typography from '@wui/basics/typography';
 
 import { useToggledState, useGlobalContext } from '@@/hooks';
 import NavMenu from '@@/components/NavMenu';
-import { SUPPORT_PHONE_NUMBER } from '@@/constants';
+import { SUPPORT_PHONE_NUMBER } from '@@/utils/constants';
```
Comment (markrawls) [inline frontend/components/Nav.js:22]: The '@@/' is no longer necessary in the new version of `NextJS` with the `jsconfig.json` file that we have

File: frontend/jest.config.js (hunk 1 above)
Comment (markrawls) [reply frontend/jest.config.js:1]: In my opinion we should, but I am not inherently correct

**Review by markrawls — CHANGES_REQUESTED** [review]

**Review by markrawls — APPROVED** [review]

**Pull request merged**